htmresearch/regions/LanguageSensor.py

```diff
@@ -104,6 +106,13 @@ def getSpec(cls):
       },
       "inputs":{},
       "parameters":{
+        "cacheRoot":{
```

Comment:
I don't understand why you need this as a parameter. The encoder is being created outside of this region and set explicitly.